packages/browser/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -81,23 +81,25 @@ export default class OidcLoginHandler implements ILoginHandler {
       options.oidcIssuer as URL
     );
 
+    const dynamicClientRegistration = await this.clientRegistrar.getClient(
```

Comment:
Is this what's needed for DCR? (i.e. it was already implemented - this hasn't changed?) If so, it still needs to be moved into oidc-dpop-client-browser.

Reply:
Yeah, absolutely - this will be moved as part of the DCR tickets (but for simple Auth Code Flow using our Broker and ESS, I needed DCR - and it was so easy to do I just reused the existing imp (i.e. 'ClientRegistrar').